libraries/AP_HAL_ChibiOS/hwdef/CubePilot-PPPGW/hwdef.dat

```diff
@@ -1,16 +1,14 @@
 include ../CubePilot-CANMod/hwdef.dat
 
 # we need RTS/CTS for the PPP link
-undef PD14
-undef PD13
 undef PE0
 undef PE1
 
 # need to use UART8 to get RTS/CTS
 PE1 UART8_TX UART8
 PE0 UART8_RX UART8
-PD13 UART8_RTS UART8
-PD14 UART8_CTS UART8
+PA10 UART8_RTS_GPIO UART8
```

Comment:
RTS is always a normal GPIO (the STM32 has a bug in handling RTS in the uart peripheral, so we avoid it)